src/Middleware/RequestThrottling/src/Internal/RequestThrottlingLog.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.RequestThrottling.Internal
+{
+    internal static class RequestThrottlingLog
+    {
+        private static readonly Action<ILogger, int, Exception> _requestEnqueued =
+            LoggerMessage.Define<int>(LogLevel.Debug, new EventId(1, "Request Enqueued"), "Server is busy; request queued in middleware. Current queue length: {queuedRequests}.");
```

Comment:
```suggestion
            LoggerMessage.Define<int>(LogLevel.Debug, new EventId(1, "Request Enqueued"), "Concurrent request limit reached, queuing request. Current queue length: {queuedRequests}.");
```

Let's clarify what "busy" means here. It means the concurrent request limit was reached.